code/src/components/Styles.scss

```diff
@@ -0,0 +1,185 @@
+/*** VARIABLES ***/
+
+$gray: #f2f0f0;
+$black: #000;
+$pink: #f4acac;
+$light-pink: #ffd5cd;
+$red: red;
+$green: green;
```

Comment:
This was neat! So, you define colors and then you implement them.

A comment is that I think you are supposed to use specify color via `#....` instead of writing red/gren